Toolset/libraries/revideextensionlibrary.livecodescript

```diff
@@ -673,10 +682,18 @@ private function __extensionPropertyInfoFromManifest pId, pManifestPath
       __SetMetadata tMetadataKey, tMetadataValue, tDataA
    end repeat
    
-   repeat for each line tProperty in idePropertyNames()
-      if tDataA[tProperty] is not empty then
-         union tDataA[tProperty] with revIDEPropertyInfo(tProperty)
-         put tDataA[tProperty] into tPropertyDataA[tProperty]
+   repeat for each key tProperty in tDataA
+      local tIDEPropInfoA
+      put revIDEPropertyInfo(tProperty) into tIDEPropInfoA
+      if tIDEPropInfoA is not empty then
+         # if this is a control property that we are overriding,
+         # tData might be non-empty for the property even though
+         # there was no property node in the XML
+         union tPropertyDataA[tProperty] with tDataA[tProperty]
+         
+         # Now take any default roperty info that was not specified
+         # in the manifest
+         union tPropertyDataA[tProperty] with tIDEPropInfoA
```

Comment:
Don't we have this data already from here https://github.com/livecode/livecode-ide/pull/1905/files#diff-5a54288a32499468dd2cab9f178bed1dR643

Reply:
No - these are ones for which metadata exists but no property declaration - i.e. overrides. However, we can be a bit more efficient by skipping the ones already processed